src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -89,6 +89,7 @@ def load_arguments(self, _):
 
     with self.argument_context('webapp update') as c:
         c.argument('client_affinity_enabled', help="Enables sending session affinity cookies.", arg_type=get_three_state_flag(return_label=True))
+        c.argument('https_only', help="Redirect all traffic made to an app using HTTP to HTTPS.", arg_type=get_three_state_flag(return_label=True))
```

Comment:
Is there a reason this is exposed on `webapp update` but not `webapp create`?  Does this apply to function apps?

Reply:
our webapp create doesn't support setting any properties

Reply:
there is no az functionapp update command, created the following to track & fix https://github.com/Azure/azure-cli/issues/5793